recipe_scrapers/_grouping_utils.py

```diff
@@ -37,11 +37,11 @@ def score_sentence_similarity(first: str, second: str) -> float:
     """
 
     if first == second:
-        # Indentical sentences have maximum score of 1
+        # Identical sentences have a maximum score of 1
```

Comment:
Took me a few moments to notice the difference here :smile:

Reply:
They're almost indentical 😉